pyfr/backends/cuda/driver.py

```diff
@@ -220,7 +220,8 @@ class CUDADevAlloc(_CUDABase):
 
     def __init__(self, cuda, nbytes):
         ptr = c_void_p()
-        cuda.lib.cuMemAlloc(ptr, nbytes)
+        if nbytes:
```

Comment:
This is weird and non-standard.  The low level wrappers should do what CUDA does here if nbytes of 0 is passed.

Reply:
Ok, I'll remove and add a check to cublaslt